scripts/userstyles.yml

```
@@ -275,6 +277,13 @@ userstyles:
     readme:
       app-link: https://cinny.in
     current-maintainers: [*jn-sena]
+  claude:
+    name: Claude
+    categories: [artificial_intelligence, productivity]
+    color: text
```

```suggestion
    color: peach
```
Their logo is orangish.